payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/OTPElementUI.kt

```diff
@@ -33,106 +35,113 @@ import androidx.compose.ui.text.TextRange
 import androidx.compose.ui.text.input.TextFieldValue
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.unit.dp
+import com.stripe.android.ui.core.PaymentsThemeStatic
 
 @Composable
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 fun OTPElementUI(
     enabled: Boolean,
     element: OTPElement,
-    modifier: Modifier = Modifier,
-    colors: TextFieldColors = TextFieldColors()
+    modifier: Modifier = Modifier
 ) {
     val focusManager = LocalFocusManager.current
     val focusRequester = remember { FocusRequester() }
+    PaymentsThemeStatic {
```

Comment:
I think this should be on the caller to decide, this method should just fetch the values.

Reply:
Good catch. I meant to do that and forgot :(